src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_IfStatement.cs

```diff
@@ -59,7 +59,7 @@ public override BoundNode VisitIfStatement(BoundIfStatement node)
 
                 builder.Add(new BoundConditionalGoto(rewrittenCondition.Syntax, rewrittenCondition, false, afterif));
                 builder.Add(rewrittenConsequence);
-                builder.Add(new BoundLabelStatement(syntax, afterif));
+                builder.Add(new BoundSequencePoint(syntax: null, statementOpt: new BoundLabelStatement(syntax, afterif)));
```

Comment:
The similar pattern is used below in a version with `else`. I think it might also be affected.

```C#
    public void M() 
    {        
      if (condition())
           doSomething();
      else
          doSomethingElse();    // could it look as if we execute this after "doSomething()" ?
    }
```